ui/src/Thing.elm

```diff
@@ -1,15 +1,22 @@
 module Thing exposing (Model, Msg(..), Thing, initial, update, view)
 
 import Bootstrap.Button as Button
+import Bootstrap.ButtonGroup as ButtonGroup
+import Bootstrap.CDN exposing (fontAwesome)
```

Comment:
This Bootstrap.CDN includes old version of fontAwesome (4.7.0), better include latest version (5.7.2) in Main.elm, same way we included external css. Make a fontAwesome function:
```elm
fontAwesome : Html msg
fontAwesome =
    node "link"
        [ rel "stylesheet"
        , href "https://use.fontawesome.com/releases/v5.7.2/css/all.css"
        , attribute "integrity" "sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr"
        , attribute "crossorigin" "anonymous"
        ]
        []
```
And then just add `fontAwesome` it in the list where is already `mfStylesheet`